checklist/checklist_sys.html

```diff
@@ -0,0 +1,155 @@
+<h1 id="checklist-for-tests-in-machine-learning-projects">Checklist for
+Tests in Machine Learning Projects</h1>
+<p><strong>Description</strong>: This is a comprehensive checklist for
+evaluating the data and ML pipeline based on identified testing
+strategies from experts in the field.</p>
+<h2 id="general">1 General</h2>
```

Comment:
Why does the General header have a "1"? It is not a checklist item. This is a bit confusing to me.